src/client/routes/index.js

```diff
@@ -58,6 +59,7 @@ export default (store) => {
     store.dispatch(fetchPullRequestMetadata(id))
     store.dispatch(fetchPullRequestDiscussion(id))
     store.dispatch(fetchPullRequestFilesList(id))
+    store.dispatch(fetchPullRequestFiles(id))
```

Comment:
This done intentionally because after moving fetch to higher order component to make sort of prefetch before loading diff. Loading file-by-file makes little sense to load file-by-file for showing the whole PR diff. Because the order of requests finish unpredictable and in most cases, the quickest would be the smallest files that could appear everywhere and usually at the end of the diff, while the biggest ones are still loading. Thus we making user staring at progress bars instead of starting review immediately.